internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -651,7 +651,13 @@ func expandAppConfigurationEncryption(input []interface{}) *configurationstores.
 		return nil
 	}
 
+	raw := input[0]
+	if raw == nil {
+		return nil
+	}
```

Comment:
Can you please combine the two checks into line 650 like we do everywhere else in the provider? 
```
if len(input) == 0 || input[0] == nil
```
```suggestion
```